cloudos_cli/jobs/job.py

```diff
@@ -1428,14 +1428,16 @@ def get_parent_job(self, workspace_id, folder_id, verify=True):
         else:
             return parent_job_id
 
-    def delete_job_results(self, folder_id, verify=True):
+    def delete_job_results(self, job_id, mode, verify=True):
```

Comment:
@danielboloc I'd still add a check for `mode` here. Remember that a user can use the function in their python scripts so the check would be helpful in those situations.

Reply:
Added